File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q33.py
def main():
    #Input
    num = int(input("Enter a 3 digits number: "))

    #Processing

    first_number = round(num / 100)
    second_number = round((num % 100) / 10)
    third_number = (num % 100) % 10

    inverse_number = first_number + second_number * 10 + third_number * 100

    subtraction_result = num + inverse_number

    #Output
    print(f"The result of the sum between of the entered number {num} and its inverse {inverse_number} is equal to: {subtraction_result}")


main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q33.js
import { question } from "readline-sync";
function main(){
    //Input
    const num = parseInt(question("Enter a 3 digits number: "))

    //Processing
    const first_number = Math.floor(num / 100)
    const second_number = Math.floor((num % 100) / 10)
    const third_number = (num % 100) % 10

    const inverse_number = first_number + second_number * 10 + third_number * 100

    const subtraction_result = num - inverse_number 

    //Output
    console.log(`The result of the sum between the entered number ${num} and its inverse ${inverse_number} is equal to: ${subtraction_result}`)

}

main()